pkg/operator/ceph/controller/spec.go

```diff
@@ -726,9 +726,10 @@ func HostPathRequiresPrivileged() bool {
 // PodSecurityContext detects if the pod needs privileges to run
 func PodSecurityContext() *v1.SecurityContext {
 	privileged := HostPathRequiresPrivileged()
-
+	allowPrivilegeEscalation := false
```

Comment:
If we're always setting to unprivileged, let's just test on Openshift. If that fails, we may need to instead call `PodSecurityContext()` to decide if it should be privileged.

Reply:
@travisn actually I was thinking on separating this change on another PR because it's not directly related to the mgr sidecar privileged issue. WDYT?